src/ArrayToTextTable.php

```diff
@@ -13,42 +13,57 @@ class ArrayToTextTable
      * @var array
      */
     private $data;
-    
+
+    /**
+     * @var bool
+     */
+    private $paddingRight = true;
```

Comment:
add $options config instead

Reply:
I've moved the padding inside the format array.